Never give up (#272)
skybet/skybet.github.io: _posts/2019-05-20-never-give-up.md

---
layout:     post
title:      Never give up!
date:       2019-05-20
summary:    The importance of mental wellbeing and overcoming adversity
author:     rami_alnawas
tags:       aws, certification, mental, wellbeing
category:   Community
---

> I'm absolutely ecstatic to have finally achieved the target I've set myself back in 2017. I've just passed the [AWS Certified Solutions Architect - Professional SAP-C01](https://www.certmetrics.com/amazon/public/badge.aspx?i=4&t=c&d=2019-05-18&ci=AWS00361425) scoring a 859 out of 1000, the passing score is 750.

Reviewer: ```suggestion
I'm absolutely ecstatic to have finally achieved the target I set myself back in 2017. I've just passed the [AWS Certified Solutions Architect - Professional SAP-C01](https://www.certmetrics.com/amazon/public/badge.aspx?i=4&t=c&d=2019-05-18&ci=AWS00361425) scoring 859 out of 1000; the passing score is 750.
```

> I passed the associate exam back in 2017 but due to a family bereavement I had to give myself some time to grieve, losing my father has had a profound impact on my day to day life. It's not something you can put behind you, instead you have to talk to those who love you and support each other is the only way forward.

Reviewer: ```suggestion
I passed the associate exam back in 2017 but due to a family bereavement I had to give myself some time to grieve. Losing my father has had a profound impact on my day-to-day life. It's not something you can put behind you; instead you have to talk to those who love you, and supporting each other is the only way forward.
```

With my other half's support, I took a week off to study and take the exam. I studied almost 12 hours a day. My eyes were red and dry sometimes. After a couple of days of multiple practice tests, it was finally time to go to the test centre, only to get the dreaded “Unfortunately you did not achieve the passing mark of 750”.

> I got 699, the main issue for me was time management, I ran out of time with almost 10 questions to go. I had to finish what I started and improve my time management tactics.

Reviewer: ```suggestion
I got 699. The main issue for me was time management - I ran out of time with almost 10 questions to go. I had to finish what I started and improve my time management tactics.
```

Determined to succeed, I took multiple practice test and focused on understanding the what, how and why instead of just getting the right answer. I read several white papers every day, summarised key points about when to use a particular AWS service over another and used these notes to revise at the morning of the exam day.

> The exam was very long, some questions had a scroll bar with only two minutes to answer, but I was absolutely over the moon to have been presented with the Congratulations page upon ending the test with only 2 minutes two spare.

Reviewer: ```suggestion
The exam was very long - some questions had a scroll bar with only two minutes to answer, but I was absolutely over the moon to have been presented with the congratulations page after completing the test with only 2 minutes to spare.
```
